### PR TITLE
Localization fixes

### PR DESCRIPTION
**Pull request by DrDudelsack**
Fixed localization for certain country names located in countries_l_english.yml

### CODE REVIEW

File: ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java [hunk 1]
```diff
@@ -665,8 +665,13 @@ else if (revoltName.equals("no")) {
                             output[0] = output[0].substring(3,output[0].length()-1);
 
                         }
+                        
+                        String adjName = tag.replace("_NAME","");
+                        String feuName = tag.replace("_FEUDATORY_NAME","_FEUDATORY_ADJECTIVE"); //for certain mission tags which have different formatting
```
Comment (Idhrendur) [inline ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java:685]: Wouldn't ```feuAdj``` be closer to what you're doing? better yet, use the full words: ```feudatoryAdjective```

Comment (DrDudelsack) [reply ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java:685]: I could use `feudatoryAdjective` instead, but `feuName` was quicker to type, lol, and my comment provides explanation on it's usage

Comment (Idhrendur) [reply ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java:685]: It's easy to type, but is harder to understand when reviewing or maintaining. Abbreviations are a bad idea. See Effective Java Item 68 (Google it if you don't have a copy).

File: ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java [hunk 2]
```diff
@@ -217,6 +217,22 @@ else if (qaaa.split("=")[0].equals( tab+tab+tab+"country_name" ) ) {
...
                                             qaaa = qaaa.substring(1,qaaa.length()-1);
+                                            if (qaaa.equals("CIVILWAR_FACTION_NAME")) {
+                                                int civilWarFlag = 0;
+                                                while (civilWarFlag == 0) {
+                                                    qaaa = scnr.nextLine();
+                                                    qaaa = qaaa.replace(tab,"");
+                                                    if (qaaa.split("=")[0].equals("name")) {
+                                                        qaaa = qaaa.split("=")[1];
+                                                        qaaa = qaaa.substring(1,qaaa.length()-1);
+                                                        if (qaaa.equals("CIVILWAR_FACTION_NAME")) {
```
Comment (Idhrendur) [inline ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java]: As Codacy note, this if statement is empty. Reverse the logic to make it simpler (unless this is a placeholder for logic in the if):

```
if (!qaaa.equals("CIVILWAR_FACTION_NAME")) {
  civilWarFlag = 1;
}
```

Comment (DrDudelsack) [reply ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java]: The logic has been reversed. Is PR#37 now good to merge?

**Review by Idhrendur — APPROVED** [review]

**Pull request merged**